src/Outlines.js

```diff
@@ -412,8 +412,40 @@ export default class Outlines {
 			return open + info.label + " " + info.qualifiedNumber + groups.close;
 		});
 
-		// TODO replace links to other files in the outline with local links
+		// Replace links to other files in the outline with local links
 		// (whether they are empty or not)
+		content = content.replaceAll(hrefRegex, (match, ...args) => {
```

Comment:
I think we have a `urls` field in one of the objects, right? Can we perhaps gate this behind a conditional so it's only executed when the outline actually spans multiple URLs?

Reply:
I was using a condition with `urls` a bit later, but yeah, it makes more sense to do it earlier. Thanks! Fixed.